### PR TITLE
Added notes about waiting 20 seconds after using some DDL statements

### PR DESCRIPTION
**Pull request by randomJoe211**

### CODE REVIEW

**Review by Amber1990Zhang — DISMISSED** [review]

File: docs-2.0/2.quick-start/4.nebula-graph-crud.md [hunk 1]
```diff
@@ -53,6 +53,8 @@ From the **Status** column of the table in the return message, you can see that
     CREATE SPACE <graph_space_name> [(partition_num=<partition number>, replica_factor=<replica number>), vid_type=fixed_string(<string_number>))]
     ```
 
+    > **NOTE:** After the creation of a graph space, wait 20 seconds before running the `USE` statement to use the graph space.
```
Comment (whitewum) [inline docs-2.0/2.quick-start/4.nebula-graph-crud.md:56]: and you can also use `show spaces xxx` or `describe space xxx` to ensure the specific space is created

File: docs-2.0/2.quick-start/4.nebula-graph-crud.md [hunk 2]
```diff
@@ -126,6 +128,8 @@ CREATE {TAG | EDGE} {<tag_name> | <edge_type>}(<property_name> <data_type>
 [, <property_name> <data_type> ...]);
 ```
 
+> **NOTE:** After the creation of a tag or an edge type, wait 20 seconds before inserting vertices or edges that has the tag or edge type.
```
Comment (whitewum) [inline docs-2.0/2.quick-start/4.nebula-graph-crud.md:131]: ditto

show / desc

File: docs-2.0/2.quick-start/4.nebula-graph-crud.md [hunk 3]
```diff
@@ -577,3 +581,17 @@ Got 1 rows (time spent 1235/1934 us)
...
+
+* `CREATE SPACE`
+* `CREATE TAG`
+* `CREATE EDGE`
+* `ALTER TAG`
+* `ALTER EDGE`
+* `CREATE TAG INDEX`
+* `CREATE EDGE INDEX`
+
+ The default heartbeat interval is 10 seconds. To make sure the follow-up operations work as expected, wait 20 seconds after performing the preceding operations.
```
Comment (whitewum) [inline docs-2.0/2.quick-start/4.nebula-graph-crud.md:597]: ( you can find the parameter in etc/xxxxfile ) TODO

File: docs-2.0/2.quick-start/4.nebula-graph-crud.md (hunk 1 above)
Comment (whitewum) [reply docs-2.0/2.quick-start/4.nebula-graph-crud.md:56]: besides. I'm not sure if `sleep(20)` is ok in nGQL 2.0

**Review by whitewum — APPROVED** [review]

**Review by Amber1990Zhang — APPROVED** [review]

**Pull request merged**